HPXMLtoOpenStudio/resources/EPvalidator.xml

```diff
@@ -678,6 +690,18 @@
       <sch:report role='WARN' test='number(h:AnnualCoolingEfficiency[h:Units="EER" or h:Units="CEER"]/h:Value) &lt; 8'>AnnualCoolingEfficiency[Units="EER" or Units="CEER"]/Value should typically be greater than or equal to 8.</sch:report>
     </sch:rule>
   </sch:pattern>
+  
+  <sch:pattern>
+    <sch:title>[CoolingSystemType=PTAC]</sch:title>
+    <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Systems/h:HVAC/h:HVACPlant/h:CoolingSystem[h:CoolingSystemType="packaged terminal air conditioner"]'>
+      <sch:assert role='ERROR' test='count(h:DistributionSystem) = 0'>Expected 0 element(s) for xpath: DistributionSystem</sch:assert> <!-- Future: allow ducted PTAC systems-->
+      <sch:assert role='ERROR' test='count(h:CoolingCapacity) &lt;= 1'>Expected 0 or 1 element(s) for xpath: CoolingCapacity</sch:assert>
+      <sch:assert role='ERROR' test='count(h:AnnualCoolingEfficiency[h:Units="EER" or h:Units="CEER"]/h:Value) + count(h:YearInstalled) &gt;= 1'>Expected 1 or more element(s) for xpath: AnnualCoolingEfficiency[Units="EER" or Units="CEER"]/Value | YearInstalled</sch:assert>
+      <sch:assert role='ERROR' test='count(h:SensibleHeatFraction) &lt;= 1'>Expected 0 or 1 element(s) for xpath: SensibleHeatFraction</sch:assert>
+      <sch:assert role='ERROR' test='count(h:CompressorType) &lt;= 1'>Expected 0 or 1 element(s) for xpath: CompressorType</sch:assert>
+      <sch:assert role='ERROR' test='h:CompressorType[text()="single stage"] or not(h:CompressorType)'>Expected CompressorType to be 'single stage'</sch:assert> <!-- Future: allow two speed and variable speed PTAC systems-->
```

Comment:
I suggest removing these lines (as we do for Room AC) since we don't offer different models.